client/src/Main.elm

```diff
@@ -1307,7 +1356,12 @@ update msg model =
             ( { model | router = changePage model.router url }, Cmd.none )
 
         SwitchPage page ->
-            ( model, gotoRoute model page )
+            case page of
+                Catalogue ->
+                    ( { model | vectorToAdd = Just <| Level0Vec initLevel0 }, gotoRoute model page )
```

Comment:
Huh!?

Reply:
Yeah not ideal...

The issue is the `overhangRadioRow`. If vectorToAdd is a Level0Vec, `theOverhangShape` = `allBsa1Overhangs`. If vectorToAdd is a LevelNVec, `overhangShape` depends on `currLevel1App`.

So when you switch from the "Add Level 1 page" to the vector catalog, you should be able to see all kinds of overhangs and not a subset.